definitions/npm/ramda_v0.21.x/flow_v0.28.x-v0.30.x/ramda_v0.21.x.js

```diff
@@ -553,8 +553,8 @@ declare module ramda {
 
   // TODO over
 
-  declare function path<V,A:?NestedObject<V>>(p: Array<string>, ...rest: Array<void>): (o: A) => ?V;
```

Comment:
Hmm.  They definitely can handle nullable values.  It's possible that I just accidentally copied these lines from an older version of the typedef.  Sorry about that!